core/src/Kokkos_Threads.hpp

```diff
@@ -107,11 +107,12 @@ class Threads {
   /// return asynchronously, before the functor completes.  This
   /// method does not return until all dispatched functors on this
   /// device have completed.
-  static void impl_static_fence();
-  static void impl_static_fence(const std::string& name);
+  static void impl_static_fence(
+      const std::string& name =
+          "Kokkos::Threads::impl_static_fence: Unnamed Static Fence");
```

Comment:
Note that would change the label that is currently printed...
https://github.com/kokkos/kokkos/blob/a793105178a9516077cdff37c07bdab2f8ba17b8/core/src/Threads/Kokkos_ThreadsExec.cpp#L298-L300